src/westpa/analysis/trajectories.py

```diff
@@ -255,6 +256,48 @@ def fget(walker, include_initpoint=True, atom_indices=None, sim_root='.'):
         self.segment_collector.show_progress = True
 
 
+class HDF5MDTrajectory(Trajectory):
+    """A MD trajectory stored by the HDF5 framework.
+
+    Parameters
+    ----------
+    atom_indices : list, default `None`
+
+    """
+
+    def __init__(self, atom_indices=None):
+        def fget(walker, include_initpoint=False, atom_indices=atom_indices):
+            from westpa.core.h5io import WESTIterationFile
```

Comment:
Move import to top of file per PEP 8.

Reply:
It is imported here to avoid the whole analysis module being dependent on `h5io`.

Reply:
I wouldn't worry about it--it's already done here: https://github.com/jpthompson17/westpa/blob/d738f18eb030482fcd0e5832a0b301a88baa669f/src/westpa/analysis/core.py#L7